bescort.lic

```diff
@@ -390,7 +390,13 @@ class Bescort
       path.each do |movement|
         move(movement)
         waitrt
-        fput('collect rock')
+        case bput('collect rock','You manage to collect a pile', 'The room is too cluttered')
```

Comment:
Logic has been moved and collecting in crossing-training and bescort consolidated to use it